### PR TITLE
fix broken links[HSTN-917]

### PR DESCRIPTION
**Pull request by hngerebara**
Fix broken links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,12 +68,13 @@ gruntwork-install --module-name 'ecs-scripts' --repo 'https://github.com/gruntwo
...
-module](https://github.com/gruntwork-io/script-modules/tree/master/modules/vault-ssh-helper) from the [script-modules
-repo](https://github.com/gruntwork-io/script-modules), passing two custom parameters to it:
+Install the [fail2ban
+module](https://github.com/gruntwork-io/module-security/tree/master/modules/fail2ban) from the [module-security
+repo](https://github.com/gruntwork-io/module-security), passing two custom parameters to it:
+
 
 ```
-gruntwork-install --module-name 'vault-ssh-helper' --repo 'https://github.com/gruntwork-io/script-modules' --tag '0.0.3' --module-param 'install-dir=/opt/vault-ssh-helper' --module-param 'owner=ubuntu'
+gruntwork-install --module-name `fail2ban` --repo `module-security` -module-param ban-time=3600
```
Comment (brikis98) [inline README.md]: These should be single quotes, not backticks... And the `bantime=3600` should be wrapped in single quotes too.

Comment (hngerebara) [reply README.md]: @brikis98 Done

File: README.md [hunk 2]
```diff
@@ -119,7 +120,7 @@ and then uses it to install several modules:
       "type": "shell",
       "inline": [
         "gruntwork-install --module-name 'ecs-scripts' --repo 'https://github.com/gruntwork-io/module-ecs' --tag 'v0.0.1'",
-        "gruntwork-install --module-name 'vault-ssh-helper' --repo 'https://github.com/gruntwork-io/script-modules' --tag 'v0.0.3' --module-param 'install-dir=/opt/vault-ssh-helper' --module-param 'owner=ubuntu'",
+        "gruntwork-install --module-name 'fail2ban' --repo 'https://github.com/gruntwork-io/module-security' -module-param 'ban-time=3600' --module-param 'install-dir=/opt/fail2ban' --module-param 'owner=ubuntu'",
```
Comment (brikis98) [inline README.md]: I don't think `fail2ban` supports `install-dir` or `owner` params, so remove those.

File: gruntwork-install [hunk 3]
```diff
@@ -43,10 +43,9 @@ function print_usage {
   echo "Example:"
   echo
   echo "  gruntwork-install \\"
-  echo "    --module-name 'vault-ssh-helper' \\"
-  echo "    --repo 'https://github.com/gruntwork-io/script-modules' \\"
-  echo "    --tag '~>0.0.3' \\"
-  echo "    --module-param 'install-dir=/opt/vault-ssh-helper' \\"
+  echo "    --module-name 'fail2ban' \\"
+  echo "    --repo 'https://github.com/gruntwork-io/module-security' \\"
+  echo "    --module-param 'install-dir=/opt/fail2ban' \\"
```
Comment (brikis98) [inline gruntwork-install]: Same here. It supports `ban-time`, but not `install-dir` or `owner`

**Pull request merged**